R/qenv-join.R

```diff
@@ -7,102 +7,102 @@
 #' See below for an example.
 #'
 #' There are some situations where `join()` cannot be properly performed, such as these three scenarios:
-#' \enumerate{
-#'   \item Both `qenv` objects contain an object of the same name but are not identical. \cr\cr
-#'   Example:
-#'   \preformatted{
-#'   x <- qenv(
-#'     code = c(mtcars1 = "mtcars1 <- mtcars"),
-#'     env = list2env(list(mtcars1 = mtcars))
-#'   )
-#'   y <- qenv(
-#'     code = c(mtcars1 = "mtcars1 <- mtcars['wt']"),
-#'     env = list2env(list(mtcars1 = mtcars['wt']))
-#'   )
-#'   z <- join(x, y)
-#'   # Error message will occur
-#'   }
-#'   In this example, `mtcars1` object exists in both `x` and `y` objects but the content are not identical.\cr
-#'   `mtcars1` in the `x qenv` object has more columns than `mtcars1` in the `y qenv` object (only has one column).
-#'   \item `join()` will look for identical `@id` values in both `qenv` objects.
-#'   The index position of these `@id`s must be the same to determine the evaluation order.
-#'   Otherwise, `join()` will throw an error message.\cr\cr
-#'   Example:
-#'   \preformatted{
-#'   common_q <- qenv(code = "v <- 1", env = list2env(list(v = 1)))
-#'   x <- eval_code(
-#'     common_q,
-#'     "x <- v"
-#'   )
-#'   y <- eval_code(
-#'     common_q,
-#'     "y <- v"
-#'   )
-#'   z <- eval_code(
-#'     y,
-#'     "z <- v"
-#'   )
-#'   q <- join(x, y)
-#'   join_q <- join(q, z)
-#'   # Error message will occur
+#' 1. Both `qenv` objects contain an object of the same name but are not identical.\cr\cr
+#'    Example:
```

Comment:
How about ditching the `\cr` as well?

```suggestion
#' 1. Both `qenv` objects contain an object of the same name but are not identical.
#'
#'    Example:
#'
```